tlatools/org.lamport.tlatools/test/tlc2/tool/DumpAsDotTest.dot

```diff
@@ -12,27 +12,27 @@ color="white";
 1377963776297717291 [label="/\\ b = TRUE\n/\\ x = 2",style = filled]
 7147721571019581646 [label="/\\ b = FALSE\n/\\ x = 3",style = filled]
 3881310712274735899 [label="/\\ b = TRUE\n/\\ x = 3",style = filled]
-609737673425276830 -> 8671809759910816123 [label="B",color="2",fontcolor="2"];
+609737673425276830 -> 8671809759910816123 [label="B {}",color="2",fontcolor="2"];
```

Comment:
How about suppressing the empty curly braces when an action doesn’t have any parameters?